source/_integrations/spokestack_wakeword.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: "Spokestack Wakeword"
+description: "Spokestack Wakeword documentation"
+ha_release: "0.38"
+ha_category: Voice
+ha_iot_class: "Local Polling"
+ha_config_flow: true
+ha_codeowners:
+  - '@spokestack'
+  - '@will-rice'
+ha_domain: spokestack_wakeword
+---
+
+[Spokestack](https://www.spokestack.io/) offers open-source libraries for adding a voice interface to _anything_. `spokestack_wakeword` is an integration that enables users to add a spoken keyword
+to trigger an event.
+
+If you already have an account, [log in](https://www.spokestack.io/login), otherwise you will need to [create](https://www.spokestack.io/create) one. The model download URL can be found in your [account settings](https://www.spokestack.io/account/settings).
+
+
+## Dependencies
+There are also some dependencies that should be installed before using this integration. Audio input is controlled by `pyaudio`, which has [PortAudio](http://www.portaudio.com/) as a dependency. You can install PortAudio from command line:
```

Comment:
The largest part of our user base runs the Home Assistant OS or the Home Assistant Container-based installation. How would these do this?

Reply:
I think I may have documented this the wrong way. I looked at a subset of the available [containers](https://hub.docker.com/u/homeassistant) and it looked like the two command-line installation methods were `apt` and `apk`. I thought, based on `apk`, that Home Assistant OS was Alpine based. However, looking at the [repo](https://github.com/home-assistant/operating-system) it says that it isn't based on a regular Linux distribution. I will update this section to include the container names instead of what I did.

Reply:
The point being, those users can't install additional packages.

Reply:
oh. I see the problem. are those users unable to install ffmpeg also? I looked at the [ffmpeg integration](https://www.home-assistant.io/integrations/ffmpeg/). Microphone from ffmpeg was the backup plan, but I'm not sure of a python audio package that doesn't require any C dependencies.

Reply:
@frenck I'm working on eliminating the C dependency. That seems like the best way forward.